data/sdsswork.cfg

```diff
@@ -271,69 +271,70 @@ sdR_N = $BOSS_SPECTRO_DATA_N/{mjd}/sdR-{br}{id}-{frame}.fit
 sdR_S = $BOSS_SPECTRO_DATA_S/{mjd}/sdR-{br}{id}-{frame}.fit
 
 #boss DRP paths
-spplan2d = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spplan2d-@pad_fieldid|-{mjd}.par
-spPlancomb = $BOSS_SPECTRO_REDUX/{run2d}/@pad_fieldid|@isplate|/spPlancomb-@pad_fieldid|-{mjd}.par
+spplan2d = $BOSS_SPECTRO_REDUX/{run2d}/@sptypefolder|/@fieldgrp|/@pad_fieldid|@isplate|/spPlan2d-@pad_fieldid|-{mjd}.par
```

Comment:
does this file name change break backwards compat?

Reply:
This was incorrect previously. I have updated the IPL-3 version as well